module/mempool/queue/internal/requestEntity.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import "github.com/onflow/flow-go/model/flow"
+
+var _ flow.Entity = (*RequestEntity)(nil)
+
+// RequestEntity is an internal data structure for storing entity requests.
+type RequestEntity struct {
```

Comment:
I think this is unused now, can we remove?

Reply:
https://github.com/onflow/flow-go/pull/3404/commits/c067dca68997a8a1274f2dbb80fb8deaf8e9ba6c